VMAccess/vmaccess.py

```diff
@@ -275,6 +276,16 @@ def _set_user_account_pub_key(protect_settings, hutil):
 
     # Reset ssh key with the new public key passed in or reuse old public key.
     if cert_txt:
+        # support for SSH2-compatible format for public keys in addition to OpenSSH-compatible format
+        if (cert_txt.strip().startswith(BeginSSHTag)):
+            ext_utils.set_file_contents("temp.pub", cert_txt.strip())
```

Comment:
Will writing the key to a file be a security concern? Is there any other way to pass in the key to ssh-keygen command?

Reply:
Actually, writing the key into the command line is also a concern, I think writing to file might be best way to go. Let me know if you have any other ideas.